apache.Dockerfile

```diff
@@ -10,12 +10,16 @@
 
 FROM docker.io/node:8.16.2 as builder
 
+RUN if [ "$(uname -m)" = "s390x" ]; then apt-get update; apt-get install -y phantomjs; fi
```

Comment:
hello, it looks like too hardcoded for a specific arch

maybe you need to provide a DOCKER_ARG to have skip-tests option and ignore optional

Reply:
@benoitf If my understanding is correct, passing DOCKER_ARG can impact Github Actions and will need to be taken care of with additional `docker build buildx` commands. Please let me know your thoughts.

Reply:
@benoitf are you OK to merge this PR as is? or do you want changes?